### PR TITLE
En 1553 improving of label key validation

### PR DESCRIPTION
**Pull request by eugeniosant**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: engine/src/main/resources/rest/messages.properties [hunk 1]
```diff
@@ -216,8 +216,11 @@ user.passwords.same=New password has to be different from old password
 user.status.invalid=User status should be ''active'' or ''inactive''
 
 #labels
-labelRequest.titles.required=No languages specified
+labelRequest.key.required=the key is required
+labelRequest.key.invalidSize=the key is longer than 50 characters.
+labelRequest.key.invalidCharacters=the key contains invalid characters. Only alphanumeric characters and the underscore ''_'' are allowed.
```
Comment (joewhite101) [inline engine/src/main/resources/rest/messages.properties]: Remove "the" before the word underscore.

Comment (eugeniosant) [reply engine/src/main/resources/rest/messages.properties]: Done

**Comment by entando-service** [comment]
All is well

**Pull request merged**